Sources/ArcGISToolkit/Components/Form/FormInputs/DateTimeEntry.swift

```diff
@@ -60,11 +60,17 @@ struct DateTimeEntry: View {
         }
         .padding([.bottom], elementPadding)
         .onAppear {
-            if let date = featureForm?.feature.attributes[element.fieldName] as? Date {
-                self.date = date
+            if element.value.isEmpty {
+                date = nil
+            } else {
+                date = try? Date(element.value, strategy: Date.ParseStrategy.arcGISDateParseStrategy)
```

Comment:
See the changes to the extension as well...
```suggestion
                date = try? Date(element.value, strategy: .arcGISDateParseStrategy)
```